aws-datastore/src/main/java/com/amplifyframework/datastore/DataStoreConfigurationProvider.java

```diff
@@ -15,6 +15,18 @@
 
 package com.amplifyframework.datastore;
 
+/**
+ * Implementations of this interface should provide access to a <code>DataStoreConfiguration</code>
```

Comment:
instead of `<code>DataStoreConfiguration</code>`, there is `{@link DataStoreConfiguration}` :)